content/articles/create-a-weather-app-in-node.js-using-openweathermap-api/index.md

```diff
@@ -0,0 +1,560 @@
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+
+                        <% } %>
+
+                            <% if(error !== null){ %>
+                                <p>
+                                    <%= error %>
+                                </p>
+                                <% } %>
+        </fieldset>
+        </div>
+```
+
+> **Note:** Scripts are always placed after the main contents on your page. This will allow the page content to fully load before JavaScript files are executed hence preventing errors. This is a good coding practice.
+
+#### Styling our page
+
+We shall style the static webpage generated and stored in the public folder during code exercution using our '**style.css**' file. This shall be our css fomart:
+
+```css
+body {
+    width: auto;
+    margin: 0 auto;
+    font-family: 'Open Sans', sans-serif;
+}
+
+
+/* This will format the whole fieldset content*/
+
+.container {
+    width: 80%;
+    margin: 0 auto;
+}
+
+
+/* This will format the whole fieldset content*/
+
+fieldset {
+    display: block;
+    -webkit-margin-start: 0px;
+    -webkit-margin-end: 0px;
+    -webkit-padding-before: 0em;
+    -webkit-padding-start: 0em;
+    -webkit-padding-end: 0em;
+    -webkit-padding-after: 0em;
+    border: 0px;
+    border-image-source: initial;
+    border-image-slice: initial;
+    border-image-width: initial;
+    border-image-outset: initial;
+    border-image-repeat: initial;
+    min-width: -webkit-min-content;
+    padding: 30px;
+}
+
+/* Format the input section */
+
+/* Format the input, paragraph, hover effect, focus and button */
+.ghost-input,
+p {
+    display: block;
+    font-weight: 300;
+    width: 100%;
+    font-size: 25px;
+    border: 0px;
+    outline: none;
+    width: 100%;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    color: #4b545f;
+    background: #fff;
+    font-family: Open Sans, Verdana;
+    padding: 10px 15px;
+    margin: 30px 0px;
+    -webkit-transition: all 0.1s ease-in-out;
+    -moz-transition: all 0.1s ease-in-out;
+    -ms-transition: all 0.1s ease-in-out;
+    -o-transition: all 0.1s ease-in-out;
+    transition: all 0.1s ease-in-out;
+}
+
+.ghost-input:focus {
+    border-bottom: 1px solid #ddd;
+}
+
+.ghost-button {
+    background-color: transparent;
+    border: 2px solid #ddd;
+    padding: 10px 30px;
+    width: 100%;
+    min-width: 350px;
+    -webkit-transition: all 0.1s ease-in-out;
+    -moz-transition: all 0.1s ease-in-out;
+    -ms-transition: all 0.1s ease-in-out;
+    -o-transition: all 0.1s ease-in-out;
+    transition: all 0.1s ease-in-out;
+}
+
+.ghost-button:hover {
+    border: 2px solid #515151;
+}
+
+p {
+    color: #E64A19;
+}
+```
+
+#### Application Environment virables
+
+You shall now enter your application environments constants.
```

Comment:
Also another tip, if you choose to write the article say in first person, stick to first person, do not change it in between the article. I personally advocate for the use of `You` and `Let's` to have a nice conversational tone in the article.

Reply:
Ok. Understood

Reply:
Done :rocket: